src/components/HeaderMenu.js

```diff
@@ -31,24 +33,21 @@ function Header({ token, setToken }) {
             )
             .then(() => {
                 setToken(null);
-                // localStorage.clear();
-                console.log("logout");
+                localStorage.clear();
```

Comment:
What does localStorage.clear() do? I haven't seen that before.

Reply:
I left it there by mistake .It does nothing in the code,